src/docs/firmware/manually-flashing.md

```diff
@@ -4,16 +4,32 @@
 Flashing your own firmware has the potential to brick your device. Do not do this unless you are sure you know what you are doing/have a way to recover from a bad flash. Some level of knowledge with using the Linux command line is required.
 :::
 
-1. Have your firmware rom. We will assume you already have the file you want to flash for this guide.
-2. Download flashrom, then give it execute permission.
+1. The steps below assume you are flashing an image named `coreboot.rom`; substitute it as necessary.
+
+2. Download flashrom, cbfstool, and gbb utility, decompress, and ensure they are executable:
    - For most distributions: `cd; curl -LO https://files.tree123.org/utils/x86_64/gnu/flashrom; chmod +x flashrom`
    - For musl (Alpine Linux): `cd; curl -LO https://files.tree123.org/utils/x86_64/musl/flashrom; chmod +x flashrom`
+   - `wget https://mrchromebox.tech/files/util/cbfstool.tar.gz && tar -zxf cbfstool.tar.gz && chmod +x cbfstool`
+   - `wget https://mrchromebox.tech/files/util/gbb_utility.tar.gz && tar -zxf gbb_utility.tar.gz && chmod +x gbb_utility`
```

Comment:
These require a musl version aswell

Reply:
I suppose I could compile them myself, although it might take me a few days to get a computer and set up Alpine or some other musl system on it.

Reply:
Alpine is too confusing, need help